src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PortableInfoboxObject.java

```diff
@@ -208,12 +234,26 @@ public PortableInfoboxObject isVideoCaptionVisible() {
   }
 
   public PortableInfoboxObject isHeroImageCentered() {
-    Assertion.assertEquals(images.get(0).getCssValue("text-align"), "center");
+    Assertion.assertEquals(images.get(0).getCssValue("vertical-align"), "middle");
     PageObjectLogging.log("Hero image", "is centered", true);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Hero image" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)